src/System.Reflection.Primitives/src/System/Reflection/CallingConventions.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
We should do a sweep to clean up these headers. Some of them are empty, some have (probaly outdated) email markers, some are just formatted weirdly.